js/models/adaptModel.js

```diff
@@ -753,21 +757,27 @@ export default class AdaptModel extends LockingModel {
   }
 
   setUnlockFirstLocking() {
-    const children = this.getAvailableChildModels();
+    const children = (this.get('_type') !== 'course')
```

Comment:
you could put the below into its own method and name the method `getChildren` to avoid code duplication

    ```
const children = (this.get('_type') !== 'course')
      ? this.findAncestor().findDescendantModels('pages', { where: { _isAvailable: true } })
      : this.findDescendantModels('pages', { where: { _isAvailable: true } });
```